components/brave_rewards/browser/publisher_info_database.cc

```diff
@@ -347,15 +348,26 @@ PublisherInfoDatabase::GetMediaPublisherInfo(const std::string& media_key) {
   if (!initialized)
     return info;
 
-  sql::Statement info_sql(
-      db_.GetUniqueStatement("SELECT pi.publisher_id, pi.name, pi.url, "
-                             "pi.favIcon, pi.provider, pi.excluded, ai.percent "
-                             "FROM media_publisher_info as mpi "
-                             "INNER JOIN publisher_info AS pi ON "
-                                "mpi.publisher_id = pi.publisher_id "
-                             "INNER JOIN activity_info AS ai ON "
-                                "mpi.publisher_id = ai.publisher_id "
-                             "WHERE mpi.media_key=?"));
+  std::string sql_str;
+  if (use_as_publisher_key) {
```

Comment:
This is part of regular publisher load, so we should just call that function if we need to

Reply:
Nothing should change from regular publisher load as `use_media_key` is an optional parameter. Should it still be split out to separate?